broker/plugins/kubejobs/plugin.py

```diff
@@ -182,6 +192,16 @@ def terminate_job(self):
     def stop_application(self):
         self.rds.delete("job")
 
+    """ When triggered, this function activate a specific cluster informed in 
+        the json file submitted by the user in all Asperathos components.
+    """
+    def active_cluster(self, data):
+        request_url = "http://%s:%s/submissions/cluster/%s" % (api.host, api.port, data['cluster_name'])
+        headers = {'Content-type': 'application/json'}
+        request_data = {'user': data['username'], 'password': data['password'], 'enable_auth': data['enable_auth']}
+        request_data = json.dumps(request_data)
+        requests.post(request_url, data=request_data, headers=headers)
```

Comment:
why don't you call `api.activate_cluster()` directly?

Reply:
Modification made in commit 540b521953feea4e73758f6d8173a1c58e9e65be